packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.types.ts

```diff
@@ -97,6 +97,12 @@ export interface IHoverCardProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   onCardExpand?: () => void;
 
+  /**
+   * Optional Native props (s) for the host div of HoverCard
+   */
+
+  hostNativeProps?: React.HTMLAttributes<HTMLDivElement>;
```

Comment:
We already pass these props to the card element. What is the scenario for needing to pass the native props to the outer host element? I could not find anything in the PR description nor any issue it's fixing.

Reply:
Yes we pass these props to card element but i needed for outer host why because i need to pass accessibility props for host like role and aria-label and aria expanded to resolve this issue https://office.visualstudio.com/OC/_workitems/edit/4221810

Reply:
Not sure how you use the HoverCard in your case. I assume, and correct me if I am wrong, from what you say you use a plain text wrapped by the `HoverCard` component and you don't provide a `target` prop. In that case you just need to wrap `7 more` with an html element and then wrap it with the HoverCard, make it focusable and put the accessibility attributes on that element. Not sure if adding another prop is the right solution. 

@micahgodbolt, who is leading the accessibility of controls in the core team? Would like their opinion on adding such a wide open prop that allows for so many attributes to be set.